src/nmodl.rs

```diff
@@ -384,7 +384,7 @@ fn nmodl_init_block(n: &Nmodl) -> Result<String> {
     let init = n
         .init
         .values()
-        .map(|s| s.print_to_string(2))
+        .map(|s| s.simplify().print_to_string(2))
```

Comment:
This simplification -- while needed -- can it be done outside printing, eg after adding/substituting terms?

Reply:
That should be the proper position indeed, but somehow I couldn't get that to work..

Reply:
https://github.com/thorstenhater/nmlcc/pull/23/commits/8ce1857575fe76b86aebc0a623584a7286879457